src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,59 @@
+@import '../utils/variables';
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    color: $normal-color;
```

Comment:
Ensure that `$normal-color` is defined in the imported variables file. If not, this could lead to a compilation error or unexpected styling.